MapleServer2/Tools/InventoryController.cs

```diff
@@ -182,16 +182,18 @@ public static void ExpandInventory(GameSession session, InventoryTab tab)
         {
             Inventory inventory = session.Player.Inventory;
             Wallet wallet = session.Player.Wallet;
+            long meretCost = 390;
+            short expansionAmount = 6;
 
-            if (!wallet.Meret.Modify(-390))
+            if (!wallet.Meret.Modify(-meretCost))
             {
-                if (!wallet.GameMeret.Modify(-390))
+                if (!wallet.GameMeret.Modify(-meretCost))
                 {
-                    if (!wallet.EventMeret.Modify(-390))
+                    if (!wallet.EventMeret.Modify(-meretCost))
                     {
-                        if (wallet.Meret.Amount + wallet.GameMeret.Amount + wallet.EventMeret.Amount >= 390)
+                        if (wallet.Meret.Amount + wallet.GameMeret.Amount + wallet.EventMeret.Amount >= meretCost)
```

Comment:
I wonder if we should have a method inside Wallet that handles removing merets from any of these "currencies", since unless I'm mistaken this won't be the only time we'll need to try deducting from all of them if needed

Reply:
I just tested and Meret Taxi also uses all types of merets.